doctrine-mapping.xsd

```diff
@@ -200,6 +200,13 @@
     <xs:anyAttribute namespace="##other"/>
   </xs:complexType>
 
+  <xs:simpleType name="TABLENAME" id="TABLENAME">
+    <xs:restriction base="xs:token">
+      <xs:pattern value="[-._:A-Za-z0-9\`]" id="TABLENAME.pattern">
```

Comment:
more something like that ```/[a-zA-Z_\x01-\xff\`]+/``` ?
Based on https://dev.mysql.com/doc/refman/5.7/en/identifiers.html